src/bucket/PublishQueueBuckets.h

```diff
@@ -14,19 +14,23 @@ namespace stellar
 class PublishQueueBuckets
 {
   public:
+    using storage = std::map<std::string, int>;
```

Comment:
`storage` typenames start with an uppercase. Why not just calling it `BucketCount` or something like this - `Storage` doesn't really convey anything